recipe.rb

```diff
@@ -0,0 +1,13 @@
+# package %w(go rust awscli)
```

Comment:
I imagine this isn't supposed to be here?

Reply:
... nope